engine/runtime/src/main/java/org/enso/interpreter/runtime/error/DataflowError.java

```diff
@@ -49,16 +50,19 @@ public final class DataflowError extends AbstractTruffleException implements Ens
    *
    * @param payload the user-provided value carried by the error
    * @param location the node in which the error was created
+   * @param attachFullStackTraceProfile
    * @return a new dataflow error
    */
-  public static DataflowError withDefaultTrace(State state, Object payload, Node location) {
+  public static DataflowError withDefaultTrace(
+      State state, Object payload, Node location, BranchProfile attachFullStackTraceProfile) {
     assert payload != null;
     boolean attachFullStackTrace =
         state == null
             || EnsoContext.get(location)
```

Comment:
Ops, this code will have to be rewritten according to https://github.com/enso-org/enso/pull/11075#pullrequestreview-2314364236 and https://github.com/enso-org/enso/pull/11075#discussion_r1766136152!